device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -1014,6 +1014,7 @@ private void initializeStateLatches()
         }
     }
 
+    @SuppressWarnings("SameParameterValue") // This is currently always "true", but can be "false" if required.
```

Comment:
Is there a test which we can modify with value "false"? In which case we don't have to suppress this type of warnings. This one and few above is kind of in the same category

Reply:
this is a generic method to construct an exception with an error message and isRetryable flag. I wouldn't design this to be two separate methods - one specifically retryable and one not.

Reply:
there isn't a unit test for this, no